src/components/pages/summoners/[summonerTagName]/GnimtyInfoTab/GnimtyInfoTab.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, HStack, Text, VStack } from '@chakra-ui/react';
+import { useQuery } from '@tanstack/react-query';
+import dayjs from 'dayjs';
+import 'dayjs/locale/ko';
```

Comment:
```suggestion
import 'dayjs/locale/ko';

import { Box, HStack, Text, VStack } from '@chakra-ui/react';
import { useQuery } from '@tanstack/react-query';
import dayjs from 'dayjs';
```
Side effect import는 맨 위에 있는 게 좋을 것 같습니다.